test/lint/lint-submodule.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+export LC_ALL=C
+EXIT_CODE=0
+
+CMD=$(git submodule status --recursive)
```

Comment:
```suggestion
CMD=$(git submodule status)
```

If this script reports submodules which should be _deleted_, it should not lists nested ones.

Reply:
Why? Submodule is submodule?

Reply:
If an "outer" submodule will be deleted, the nested submodule will be deleted automatically (as it resides in the subdirectory of the "outer" submodule), no?

It seems no need in "delete them" instruction to a user point to nested submodules.